README.md

```diff
@@ -76,3 +80,30 @@ Things you may want to cover:
 >  ]
 >}
 >```
+
+### Wine Show Page Data
+`GET https://weathervine-be.herokuapp.com/api/v1/wines/:api_id`
+- **Required** path params:
+  - `:api_id` - id used to access wine info from external wine API
```

Comment:
This is great to demonstrate to the users visiting our application what `id` they are to use and what it would look like in the request.